docker/docker-compose.yml

```
@@ -1,41 +1,42 @@
 version: '3'
 
 services:
-    frontend:
-        image: 'burning-okr:1.0.0-frontend'
-        build: '../frontend/.'
-        container_name: 'burning-okr-frontend'
-        depends_on:
-            - backend
+    okr-db:
+        image: postgres:9.6.16
+        hostname: burning-okr-postgres
+        env_file:
+            - postgres.env
         ports:
-            - 4200:80
+            - 5489:5432
```

The Postgres-Port is a bit interesting :-)
```suggestion
            - 5432:5432
```